pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -76,6 +77,7 @@ func (a *AgentPXEFiles) Generate(dependencies asset.Parents) error {
 
 	a.imageReader = custom
 	a.cpuArch = ignition.CPUArch
+	a.pxeBaseURL = ignition.PxeBaseURL
```

Comment:
Is there a reason to pass this down through Ignition instead of depending on the AgentConfig directly?

Reply:
Similar to the way we are doing for other fields from AgentConfig like `RendezvousIP` 🤔

Reply:
Those are actually used in the Ignition though, this isn't.